### PR TITLE
Remove billing KV arcticle

### PR DESCRIPTION
**Pull request by victoriabernard92**
- Moved all KV limits and pricing out of the KV section into about
- Created pricing article so we can delete https://support.cloudflare.com/hc/en-us/articles/360001657552-Billing-for-Cloudflare-Workers-and-Workers-KV 
- Handled all redirects and broken links to` /reference/storage/pricing` and ` /reference/storage/limits`

Changes I made from original article:https://github.com/cloudflare/workers-docs/commit/bb40c984dc47118211a345f54e60e813a5be7b8d 

All pricing info in two tables
image
fixes part of #351

### CODE REVIEW

File: content/about/pricing.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Limits 
+
+To protect our systems against abuse there are surge limits in place for domains without subscriptions. 
+
+Note only requests that hit a Worker script will be accounted for in the limits and in billing. Since Cloudflare Workers runs before the Cloudflare cache, the caching of a request still incurs costs. See definitions and behavior after a limit is hit for the free plan in the [limits article](/about/limits).
+
+| Plan         | [CPU Limit](/about/limits/#cpu-execution-time-limit) | [Daily Request Limit](/about/limits/#daily-request-limit) | [Burst Rate Limit](/about/limits/#burst-rate-limit)         | Requests/month    |
+| ------------ | --------- | ------------------- | ------------------------ | ----------------- |
+| Free         | 10ms      | 100,000             | 1000 requests per minute | 10 million        |
```
Comment (victoriabernard92) [inline content/about/pricing.md]: Assumption here, please verify: Free plan Monthly request limit still 10 million?

File: content/about/pricing.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+| Unlimited    | 50ms      | none                | none                     | 10 million        |
```
Comment (signalnerve) [inline content/about/pricing.md]: I know it's been stated at the beginning of the article, but maybe I wonder if it'd be useful to re-iterate how much the Unlimited plan is again here?

File: content/about/pricing.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## KV 
+
+Enabling Workers KV requires the Unlimited plan.
+
+| Plan                 | Storage      | Reads/month       | Writes/month      | Deletes/month     | Lists/month       |
+| -------------------- | ------------ | ----------------- | ----------------- | ----------------- | ----------------- |
+| Free                 | -            | -                 | -                 | -                 | -                 |
+| Unlimited (Included) | 1 GB         | 10 million        | 1 million         | 1 million         | 1 million         |
```
Comment (signalnerve) [inline content/about/pricing.md]: Same Q as above

File: content/templates/pages/rewrite_links_html.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+hidden: true
+---
+
+{{< template-page "rewrite_links_html" >}}
```
Comment (signalnerve) [inline content/templates/pages/rewrite_links_html.md]: Meant to be in a different PR?

Comment (victoriabernard92) [reply content/templates/pages/rewrite_links_html.md]: The docs wouldn't build without this change 🤷‍♂

File: content/about/pricing.md (hunk 2 above)
Comment (victoriabernard92) [reply content/about/pricing.md]: Good point that is should be easy to spot the price of Unlimited. 
Since there are details with the plan (it's per account per month and doesn't apply to other plans) I made it more clear where to find the price by anding a name section `## Unlimited` and bolded the price.

File: content/templates/pages/rewrite_links_html.md (hunk 4 above)
Comment (victoriabernard92) [reply content/templates/pages/rewrite_links_html.md]: You're right I want ahead and fixed this on https://github.com/cloudflare/workers-docs/pull/624

File: content/about/limits.md [hunk 5]
```diff
@@ -1,20 +1,23 @@
...
+  * [Burst Rate Limit](#burst-rate-limit)
+  * [Daily Request Limit](#daily-request-limit)
+    + [Fail Open](#fail-open)
+    + [Fail Closed](#fail-closed)
+- [CPU/Execution Time Limit](#cpu-execution-time-limit)
+- [Memory](#memory)
+- [Subrequests](#subrequests)
+  * [Can a Workers script make subrequests to load other sites on the Internet?](#can-a-workers-script-make-subrequests-to-load-other-sites-on-the-internet-)
+  * [How many subrequests can I make?](#how-many-subrequests-can-i-make-)
+  * [Can I make a subrequest after my Workers has responded to the user?](#can-i-make-a-subrequest-after-my-workers-has-responded-to-the-user-)
```
Comment (ispivey) [inline content/about/limits.md]: Just realized there's a typo in the title of this section: s/Workers/Worker/

File: content/about/limits.md [hunk 6]
```diff
@@ -92,3 +95,25 @@ While handling a request, each Worker script is allowed to have up to six connec
 Once a Worker has six connections open, it can still attempt to open additional connections. However, these attempts are put in a pending queue - the connections won't be actually be initiated until one of the currently open connections has closed. Since earlier connections can delay later ones, if a Worker tries to make many simultaneous subrequests, its later subrequests may appear to take longer to start.
 
 If the system detects that a Worker is deadlocked on open connections - for instance, if the Worker has pending connection attempts but has no in-progress reads or writes on the connections that it already has open - then the least-recently-used open connection will be canceled to unblock the Worker. If the Worker later attempts to use a canceled connection, an exception will be thrown. These exceptions should rarely occur in practice, though, since it's uncommon for a Worker to open a connection that it doesn't have an immediate use for.
+
+# KV 
+
+After subscriptiont to a Workers Unlimited plan, KV is enabled. Workers KV supports:
```
Comment (ispivey) [inline content/about/limits.md]:
```suggestion
After subscription to a Workers Unlimited plan, KV is enabled. Workers KV supports:
```

File: content/about/limits.md [hunk 7]
```diff
@@ -1,20 +1,23 @@
...
 ---
-
-## Plans
-
-Workers plans are separate from any Cloudflare plan (Free, Professional, Business) you may have. All users can access the Workers free plan subject to its [request limits](#request-limits). Upgrading to the Unlimited (Paid) plan lifts the request limits and increases CPU time limits.
-
-| Plan             | CPU Limit | Daily Request Limit | Burst Rate Limit         |
-| ---------------- | --------- | ------------------- | ------------------------ |
-| Free             | 10ms      | 100,000             | 1000 requests per minute |
-| Unlimited (Paid) | 50ms      | none                | none                     |
```
Comment (ispivey) [inline content/about/limits.md]: I'd consider keeping this table on this page in a prominent position. This is an easy-to-scan summary of the most important limits.

File: content/about/pricing.md (hunk 1 above)
Comment (ispivey) [reply content/about/pricing.md]: There's no monthly request limit for either the free or Unlimited plan.

File: content/about/pricing.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+## Limits 
+
+To protect our systems against abuse there are surge limits in place for domains without subscriptions. 
+
+Note only requests that hit a Worker script will be accounted for in the limits and in billing. Since Cloudflare Workers runs before the Cloudflare cache, the caching of a request still incurs costs. See definitions and behavior after a limit is hit for the free plan in the [limits article](/about/limits).
+
+| Plan         | [CPU Limit](/about/limits/#cpu-execution-time-limit) | [Daily Request Limit](/about/limits/#daily-request-limit) | [Burst Rate Limit](/about/limits/#burst-rate-limit)         | Requests/month    |
+| ------------ | --------- | ------------------- | ------------------------ | ----------------- |
+| Free         | 10ms      | 100,000             | 1000 requests per minute | 10 million        |
+| Unlimited    | 50ms      | none                | none                     | 10 million        |
```
Comment (ispivey) [inline content/about/pricing.md]: To me, this says that the Unlimited plan has a limit of 10 million requests/month. I understand that the line below is meant to indicate that you pay for requests above 10 million, but it may be confusing to readers. I think it might be worth decoupling the _limits_ from the _pricing_ and having separate tables for those two things.

In fact, if you take my earlier suggestion to leave the "limits-by-plan" table in `limits.md`, this table could be devoted entirely to pricing, and thereby omit the Free plan and become less complex.

File: content/about/pricing.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
+---
+title: Pricing
+weight: 4
+---
+
+Workers is now free to enable for all and are subject to some limits. 
+
+## Unlimited 
+To avoid the limits and enable KV, you can subscribe to the Workers Unlimited Plan for **$5 USD per month for a account**. Workers Unlimited plan is separate from any Cloudflare plan (Free, Professional, Business) you may have.
```
Comment (ispivey) [inline content/about/pricing.md]: Do we need to specify that this is "for an account" or can we simply say that you can subscribe "for $5 USD per month" ? The latter seems simpler.

Also, consider starting the second sentence, "The Workers Unlimited plan", as that's how you referred to it in the previous sentence.

Consider "any _other_ Cloudflare plan".

File: content/about/pricing.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+If you are an Enterprise customer, reach out to your account team to confirm pricing details.
+
+## Limits 
+
+To protect our systems against abuse there are surge limits in place for domains without subscriptions.
```
Comment (ispivey) [inline content/about/pricing.md]: s/domains/accounts/ ?

Also, rather than "surge limits", consider using language that maps directly to the language in the tables below, e.g. "burst limits".

File: content/about/pricing.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+## KV 
+
+Enabling Workers KV requires the Unlimited plan.
+
+| Plan                 | Storage      | Reads/month       | Writes/month      | Deletes/month     | Lists/month       |
+| -------------------- | ------------ | ----------------- | ----------------- | ----------------- | ----------------- |
+| Free                 | -            | -                 | -                 | -                 | -                 |
+| Unlimited (Included) | 1 GB         | 10 million        | 1 million         | 1 million         | 1 million         |
+| Additional*          | $0.50 per GB | $0.50 per million | $5.00 per million | $5.00 per million | $5.00 per million |
```
Comment (ispivey) [inline content/about/pricing.md]: Consider something explicit like "Price beyond the included quota" – that would eliminate the need for the asterisk and footnote below.

File: content/about/pricing.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
+---
+title: Pricing
+weight: 4
+---
+
+Workers is now free to enable for all and are subject to some limits.
```
Comment (ispivey) [inline content/about/pricing.md]: "all, and is" ?

File: content/about/limits.md [hunk 13]
```diff
@@ -92,3 +95,25 @@ While handling a request, each Worker script is allowed to have up to six connec
 Once a Worker has six connections open, it can still attempt to open additional connections. However, these attempts are put in a pending queue - the connections won't be actually be initiated until one of the currently open connections has closed. Since earlier connections can delay later ones, if a Worker tries to make many simultaneous subrequests, its later subrequests may appear to take longer to start.
 
 If the system detects that a Worker is deadlocked on open connections - for instance, if the Worker has pending connection attempts but has no in-progress reads or writes on the connections that it already has open - then the least-recently-used open connection will be canceled to unblock the Worker. If the Worker later attempts to use a canceled connection, an exception will be thrown. These exceptions should rarely occur in practice, though, since it's uncommon for a Worker to open a connection that it doesn't have an immediate use for.
+
+# KV 
+
+After subscriptiont to a Workers Unlimited plan, KV is enabled. Workers KV supports:
+
+- Up to 100 Namespaces per account
+- Unlimited keys per Namespace
```
Comment (ispivey) [inline content/about/limits.md]: Here and above, I'd suggest lower-case "namespace", as that appears to be the convention followed elsewhere in the docs: https://developers.cloudflare.com/workers/reference/storage/namespaces/

File: content/about/pricing.md [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+## Unlimited 
+To avoid the limits and enable KV, you can subscribe to the Workers Unlimited Plan for **$5 USD per month for a account**. Workers Unlimited plan is separate from any Cloudflare plan (Free, Professional, Business) you may have.
+
+If you are an Enterprise customer, reach out to your account team to confirm pricing details.
+
+## Limits 
+
+To protect our systems against abuse there are surge limits in place for domains without subscriptions. 
+
+Note only requests that hit a Worker script will be accounted for in the limits and in billing. Since Cloudflare Workers runs before the Cloudflare cache, the caching of a request still incurs costs. See definitions and behavior after a limit is hit for the free plan in the [limits article](/about/limits).
```
Comment (ispivey) [inline content/about/pricing.md]: Instead of "will be accounted for in the limits and in billing," consider "will count against your limits and your bill," which is more direct.

"the caching of a request still incurs costs" is unclear to me. Is there a way to be more explicit about what you mean?

I think you can remove "for the free plan" – that article also describes limits on the Unlimited plan, like CPU, memory usage, subrequests, etc.
